pallets/delegation/src/lib.rs

```diff
@@ -306,7 +305,7 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		#[cfg(feature = "try-runtime")]
-		fn try_state(_n: BlockNumberFor<T>) -> Result<(), &'static str> {
+		fn try_state(_n: BlockNumberFor<T>) -> Result<(), sp_runtime::TryRuntimeError> {
```

Comment:
You seem to mix the direct use of `sp_runtime::TryRuntimeError` and `use sp_runtime::TryRuntimeError`. If used more than once in a module, it's nice to have it imported at the top of the file.